Cargo.toml

```diff
@@ -36,11 +33,11 @@ utils = []
 byteorder = "1.3"
 log = "0.4"
 flate2 = "1.0"
-bitflags = "1.2"
+bitflags = "2.4.2"
 nohash = "0.2"
-image = { version = "0.24", default-features = false }
+image = { version = "0.24.8", default-features = false }
```

Comment:
What's the reason for including the minor version here? `cargo` will already choose the latest version available.

In particular, since `image` is a public dependency it will make it incompatible with anyone using `0.24.7` (for whatever reason).

Reply:
done :)